install.sh

```diff
@@ -28,7 +28,13 @@ if [ ! -d "$CONF_PATH" ]; then
     mkdir -p $CONF_PATH
 fi
 
-# copy anadi.sh in $HOME/bin as anadi
-cp anadi.sh /usr/local/bin/anadi
+# It would be nice to be able to chose where to install the script
+# Default can still be $HOME/local/bin to avoid using sudo
+DEST_PATH=$HOME/.local/bin
```

Comment:
There is a typo in the comment,_$HOME/local/bin_ should be _$HOME/.local/bin_.
Not every Unix-like O.S. like  Linux and OSX have the path (i.e. my OSX haven't it). 
The script should check its existences and create it if missing.

Reply:
> There is a typo in the comment,$HOME/local/bin should be $HOME/.local/bin.
Thanks, I'll fix it

> The script should check its existences and create it if missing.
Good point, do you know it there is a standard `local bin` location for OSX?

Reply:
I've searched online and check on my OSX but there is no standard path and usually users rely on _/usr/local/bin_.
In previous version of the tool I was using _$HOME/.local/bin_ but  I didn't like to ha ve a different behavior between Linux and OSX.
Probably the best option is to use _$HOME/.local/bin_ for Linux OS and _/users/local/bin_ for OSX and check for usuperuser-privileges for OSX.

Any idea ?

Reply:
So it seems that every tool does whatever they want :sweat_smile: for example pyenv puts the binary in `$HOME/.pyenv/bin`.
I'd really like to get rid of the super-user privileges, at least for whom is just trying the tool for the first time, I'll try to find a better way to handle Linux and OSX.

Reply:
yes anyone do whatever it want, i.e. brew uses _/opt/brew/bin/_ :D.

One not so bad behavior can be let the Linux users use _$HOME/.local/bin/_ without superuser-provileges and for OSX we can choose if:
    - use _/users/local/bin_, with superuser-privileges
    - use _$HOME/.local/bin_ , without superuser-privileges BUT they must add that path on PATH env var.